packages/common/src/feedback/_internal/FeedbackUiSchemaSettings.ts

```diff
@@ -0,0 +1,44 @@
+import { IArcGISContext } from "../../ArcGISContext";
+import { IUiSchema } from "../../core/schemas/types";
+import { EntityEditorOptions } from "../../core/schemas/internal/EditorOptions";
+
+/**
+ * @private
+ * settings uiSchema for Hub Feedback - this
+ * defines how the schema properties should be
+ * rendered in the Feedback settings experience
+ */
+export const buildUiSchema = async (
+  i18nScope: string,
+  options: EntityEditorOptions,
+  context: IArcGISContext
+): Promise<IUiSchema> => {
+  return {
+    type: "Layout",
+    elements: [
+      {
+        type: "Section",
```

Comment:
We'll likely want to add a `rule` here to conditionally render this section when `hasMapQuestion` from the schema is `true`. This will likely look something like:

```ts
        rule: {
          effect: UiSchemaRuleEffects.SHOW,
          condition: {
            scope: '/properties/hasMapQuestion',
            schema: { const: true }
          }
        },
```

Disclaimer: I've not worked with rules yet and have not tested ^. You can see some other examples of rule definitions [here](https://github.com/Esri/hub.js/blob/master/packages/common/src/core/schemas/internal/metrics/StatCardUiSchema.ts#L329)